plugin.xml

```diff
@@ -4,7 +4,7 @@
     <description>TestFairy SDK plugin for Cordova</description>
     <license>Apache 2.0 License</license>
     <engines>
-        <engine name="cordova" version=">=3.2.0" />
+        <engine name="cordova" version=">=9.0.0" />
```

Comment:
bumping the cordova-cli version is required to use the new `<podspec>` tag. considering that cordova-cli@9 was [released in 2019](https://github.com/apache/cordova-cli/blob/master/RELEASENOTES.md#900-mar-20-2019), i think this should be ok